test/xapi/track_test.rb

```diff
@@ -24,27 +24,11 @@ def test_default_track
     assert_equal slugs, track.slugs
 
     # default test pattern
-    files = [
-      "file.ext",
-      "test.ext",
-      "file_test.ext",
-      "TestFile.ext",
-      "spec.ext",
-      "sub/test/file.ext",
-      "sub/TestDir/file.ext",
-    ]
-    assert_equal %w(file.ext spec.ext), files.reject {|f|
-      f =~ track.test_pattern
-    }
+    assert_equal(/test/i, track.test_pattern)
```

Comment:
The current test tests that:
 * track has a method `test_pattern`
 * `test_pattern` has a default value which is a specific regex.

It's a useful test because it tests that there *is* a default value.

Are you suggesting that the behaviour we need to test for is that a (currently hardcoded) regex will be equivalent to any of the (almost infinite) regexes that match(or not) that array? 
Why is that a useful test? Why are those values meaningful?

Xapi::Track never uses the test_pattern itself so I suspect the behaviour you want to test lives in another object, and the 'these filenames get filtered correctly' test should be there.